### ISSUE TITLE
Stuck between pages

### ISSUE THREAD

**Issue by edbond** [TheSeamau5/TabbedPages#1]
Sometimes it stuck between pages. To reproduce click mouse when pages moving.
Chrome, Linux.

![image](https://cloud.githubusercontent.com/assets/1044/8803495/d5cf7266-2fcf-11e5-8841-a6f62ed61a6b.png)

**Comment by TheSeamau5** [comment]
Yeah, I've noticed it. I have yet to understand the exact reason. Thanks for the report.
